src/main/java/com/google/devtools/build/lib/bazel/rules/ninja/parser/NinjaParserStep.java

```diff
@@ -181,7 +182,7 @@ public NinjaRule parseNinjaRule() throws GenericParsingException {
 
     parseExpected(NinjaToken.NEWLINE);
     while (parseIndentOrFinishDeclaration()) {
-      String variableName = asString(parseExpected(NinjaToken.IDENTIFIER));
+      String variableName = asString(parseExpected(ImmutableSet.of(NinjaToken.IDENTIFIER, NinjaToken.POOL)));
```

Comment:
Why is "pool" treated differently from an identifier?

When going through the existing code to better understand this lexer, I happened upon NinjaLexer.interpretPoolAsVariable()

So it looks like there's already some toggleable logic for treating "pool" as an identifier, no?

Reply:
NinjaToken.POOL was created together with NinjaToken.IDENTIFIER. Maybe @irengrig had plans for specializing logic with the `pool` token? As far as I can tell, `NinjaToken.POOL` is not used specially anywhere else other than NinjaParserStep.